src/components/map/map.tsx

```diff
@@ -29,7 +29,8 @@ const RECOMMENDED_MAX_ZOOM = 20;
   tag: "gx-map"
 })
 export class Map implements GxComponent {
-  private watchPositionId: number;
+  private centerCoords: string;
```

Comment:
Why do we have `centerCoords` and `center` properties. It seems both of them serve the same purpose. Could we refactor the code to only use `center`?
You can address this in another PR, if you prefer.